client/ayon_nuke/api/lib.py

```diff
@@ -593,10 +593,13 @@ def compat_prefixed(knob_name):
             if not knob_name:
                 # Ignore unnamed knob
                 continue
-
-            knob_type = nuke.knob(knob.fullyQualifiedName(), type=True)
-            value = knob.value()
-
+            try:
+                knob_type = nuke.knob(knob.fullyQualifiedName(), type=True)
+                value = knob.value()
+            except Exception:
+                log.debug(
+                    f"Error in knob {knob_name}, node {node['name'].value()}")
+                continue
```

Comment:
This is a bit of a puzzle to be honest. I am not sure how this was blocking the processing. The code is just getting knob type and value. This seems to be redundant.

Reply:
Nuke sometimes breaks on the knob type line. This can block opening Nuke scripts.

Reply:
Unfortunatelly, I can't find any examples stached for this; it only happens few times in few thousands shots.